src/Controller/OrderController.php

```diff
@@ -247,4 +245,47 @@ public function getUnzerCompanyTypes(): array
         }
         return $this->companyTypes;
     }
+
+    /**
+     * execute Unzer defined via getExecuteFnc
+     */
+    public function executeoscunzer(): ?string
```

Comment:
fände camel case lesbarer

Reply:
Hm. Das ist eine Methode die als fnc gestartet wird. Im Core werden diese Methoden alle in "klein" geschrieben. Möglicherweise ist es egal. Ich habe mich hier an den Core gehalten. Könnte ich aber ausprobieren ...